### DISCUSSION TITLE
Is there a way that i can delete all the kata's i've done and start again from 0 kyu?

### DISCUSSION THREAD

**Discussion by HabadOlad** [codewars/codewars.com#3111] [Q&A]
Is there a way that i can delete all the kata's i've done and start again from 0 kyu/or be able to restart all the katas without deleting my account? 
- i did most of them with the help of a friend/youtube and ChatGPT i would like to start again from scratch so i can really asses where my skills are?

**Comment by ejini6969** [comment]
Delete your account and open a new one with another email address or using Github. Also, bear in mind that ChatGPT should be used as a tool in helping you to learn, not a tool for plagiarizing to solve tons of katas (it is eligible for a ban if such behavior is detected by mods) Happy Coding ! ! ! ^^

**Reply by hobovsky** [reply]
If the original account is deleted, then the email and user name can be reused for the new account.
